ddl/ddl_api.go

```diff
@@ -5448,6 +5448,7 @@ func (d *ddl) UnlockTables(ctx sessionctx.Context, unlockTables []model.TableLoc
 	err := d.doDDLJob(ctx, job)
 	if err == nil {
 		ctx.ReleaseAllTableLocks()
+		ctx.GetStore().GetMemCache().Release()
```

Comment:
Will it only release one tidb-server?

Reply:
Yes, It will only release the cache in the current tidb-server.